content/blog/modeling-software-backstage/index.md

```diff
@@ -0,0 +1,203 @@
+
+![a 4th box has been added to represent an API. It is connected to passenger-backend via an arrow](./api.png)
+
+APIs are represented in Backstage using the same three properties as components.
+
+```yaml
+kind: API
+type: openapi
+name: passenger-api
+```
+
+The `type` specifies the specification language you are using to describe your API. We've specified OpenAPI here but others like GraphQL and gPRC are supported.
+
+Once we have defined the API, we can indicate that the Passenger Backend service exposes it.
+
+```yaml
+kind: Component
+type: service
+name: passenger-backend
+providesApi:
+  - passenger-api
+```
+
+Once we have indicated this relationship, we can show it off in the Backstage UI by adding the `EntityProvidedApisCard`. We would typically add this card to a tab on the Passenger Backend component so that people can look that component up in the catalog in order to read its API definition.
+
+![a table element with text showing the API which is provided by the passenger-backend. The description, lifecycle and other attributes are there.](./provided-apis-card.png)
+
+## Combining things into systems
+
+Together, the Passenger Backend and Passenger API make up a logical system. They are a group of entities with a well defined purpose, providing and managing information on passengers.
+
+![A box has been added above everything else to indicate a hierarchy. Passenger backend and the API have arrows up to the new box.](./one-system.png)
+
+To represent them as a logical group in Backstage, we can define a system. Systems don't have types, they are just systems.
+
+```yaml
+kind: System
+name: passengers
+```
+
+We can declare that the Passenger API and the Passenger Backend are part of the system by adding the system property to their definitions.
+
+```yaml
+kind: Component
+type: service
+name: passenger-backend
+system: passengers
+providesApi:
+  - passenger-api
+```
+
+```yaml
+kind: API
+type: openapi
+name: passengers
+system: passenger-backend
+```
+
+Once the system exists in Backstage, it will get it's own page in the UI where we can represent its relationships. For example we can add the `EntityHasApisCard` to see the APIs which are part of this system.
+
+![A table listing the one API which is part of the system](./entity-has-apis-card.png)
+
+Similarly, we can add the EntityHasComponentsCard to see the components which are part of the system.
+
+![A table listing the one component which is part of the system.](./entity-has-components-card.png)
+
+It's important to note that the Core Queueing Library and the Core Auth Module are not considered to be part of the Passengers system. This is because they are shared libraries which are used in a large number of components throughout the org. The are probably owned and developed by a different organization, within our ride-sharing company.
```

Comment:
`The are probably owned and developed by a different organization, within our ride-sharing company.`

Maybe another for than organization here. Team? BU?